core/modules/layout/layout.module

```diff
@@ -1678,15 +1799,15 @@ function layout_get_block_info($block_module = NULL, $block_delta = NULL) {
       return $block_info['modules'][$block_module][$block_delta];
     }
     else {
-      return FALSE;
+      return array();
```

Comment:
I'm guessing this was done for consistency with the docblock? I think we should update the docblock to be `@return array|false` rather than changing the actual return value.